projects/plugins/jetpack/extensions/blocks/seo/index.js

```diff
@@ -13,24 +14,28 @@ export const name = 'seo';
 export const settings = {
 	render: function JetpackSEODescriptionPanel() {
 		const generalPanelProps = {
-			title: __( 'SEO Description', 'jetpack' ),
 			icon: <JetpackLogo showText={ false } height={ 16 } logoColor="#1E1E1E" />,
 		};
 
 		const prePublishPanelProps = {
 			...generalPanelProps,
+			title: __( 'SEO Description', 'jetpack' ),
 			id: 'seo-title',
 		};
 
 		return (
 			<Fragment>
 				<JetpackPluginSidebar>
-					<PanelBody { ...generalPanelProps }>
-						<SeoPanel />
+					<PanelBody { ...generalPanelProps } title={ __( 'SEO Title', 'jetpack' ) }>
+						<SeoTitlePanel />
+					</PanelBody>
+					<PanelBody { ...generalPanelProps } title={ __( 'SEO Description', 'jetpack' ) }>
+						<SeoDescriptionPanel />
 					</PanelBody>
 				</JetpackPluginSidebar>
+				{ /* TODO: Add Seo Title to PrePublishPanel */ }
```

Comment:
This looks like it would be a quick add - thoughts on going ahead and taking care of this TODO? If we do not tackle in this iteration, let's create an issue instead of leaving the TODO comment here.

Reply:
I agree, we've run out of time during the hack day, but it shouldn't be an issue to add it now.